sdk/documentintelligence/Azure.AI.DocumentIntelligence/MigrationGuide.md

```diff
@@ -0,0 +1,515 @@
+- **Split options:** in previous API versions, the document splitting and classification operation always tried to split the input file into multiple documents. To enable a wider set of scenarios, `ClassifyDocument` now supports a `split` parameter. The following values are supported:
+  - `Auto`: let the service determine where to split.
+  - `None`: the entire file is treated as a single document. No splitting is performed.
+  - `PerPage`: each page is treated as a separate document. Each empty page is kept as its own document.
+
+The table below describes the relationship of each client and its supported API version(s):
+
+Package|API version|Supported clients
+|-|-|-
+|Azure.AI.DocumentIntelligence | 2023-10-31-preview | DocumentIntelligenceClient and DocumentIntelligenceAdministrationClient
+|Azure.AI.FormRecognizer |2023-07-31 | DocumentAnalysisClient and DocumentModelAdministrationClient
+|Azure.AI.FormRecognizer |2022-08-31 | DocumentAnalysisClient and DocumentModelAdministrationClient
+|Azure.AI.FormRecognizer |2.1 | FormRecognizerClient and FormTrainingClient
+|Azure.AI.FormRecognizer |2.0 | FormRecognizerClient and FormTrainingClient
+
+Please refer to the [README][readme] for more information on these new clients.
+
+## Important changes
+
+### Rebranding
+Some terminology has changed to reflect the enhanced capabilities of the latest service APIs. The "Azure Cognitive Services Form Recognizer" service has been rebranded to "Azure AI Document Intelligence", reflecting the fact that it is capable of much more than simple recognition, and is not limited to documents that are "forms". Similarly, we've made the following broad changes to the terminology used throughout the SDK:
+
+- The namespace `Azure.AI.DocumentIntelligence` has replaced `Azure.AI.FormRecognizer.DocumentAnalysis`.
+- The client `DocumentIntelligenceClient` has replaced `DocumentAnalysisClient`.
+- The client `DocumentIntelligenceAdministrationClient` has replaced `DocumentModelAdministrationClient`.
+- The type `AzureAIDocumentIntelligenceClientOptions` has replaced `DocumentAnalysisClientOptions`.
+- The type `AIDocumentIntelligenceModelFactory` has replaced `DocumentAnalysisModelFactory`.
+
+### Client usage
+
+We continue to support API key and AAD authentication methods when creating the clients. Below are the differences between the two versions:
+
+- In `Azure.AI.DocumentIntelligence`, we have `DocumentIntelligenceClient` and `DocumentIntelligenceAdministrationClient` which support API version `2023-10-31-preview` and higher.
+- Some client methods have been renamed. See the [CHANGELOG][changelog] for an exhaustive list of changes.
+
+Creating new clients in `Azure.AI.FormRecognizer`:
+```C#
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var credential = new AzureKeyCredential(apiKey);
+
```

Comment:
Should this not be using older FormRecognizerClient?

Reply:
We're comparing `DocumentIntelligenceClient` (from the new `Azure.AI.DocumentIntelligence` library) with `DocumentAnalysisClient` (from the old `Azure.AI.FormRecognizer` library).